Antidote/CallsManager.m

```diff
@@ -75,6 +75,7 @@ - (instancetype)init
 
 - (void)dealloc
```

Comment:
Log in init will be useful too. As CallsManager can be killed and recreated it would be useful to track those events.